reana_server/config.py

```diff
@@ -332,6 +332,12 @@ def _get_rate_limit(env_variable: str, default: str) -> str:
 # Email configuration
 # ===================
 ADMIN_EMAIL = os.getenv("REANA_EMAIL_SENDER", "CHANGE_ME")
+REANA_NOTIFICATIONS_ENABLED = bool(
+    strtobool(os.getenv("REANA_NOTIFICATIONS_ENALBED", "False"))
+)
+REANA_NOTIFICATIONS_EMAIL_RECEIVER = os.getenv(
+    "REANA_NOTIFICATIONS_EMAIL_RECEIVER", "CHANGE_ME"
+)
```

Comment:
What do you think about putting these variables in reana-commons in `email.py`?

Also, what do you think about using `REANA_EMAIL_RECEIVER` instead of `REANA_NOTIFICATIONS_EMAIL_RECEIVER`, to be consistent with `REANA_EMAIL_SENDER`?

Finally, let's also remove `ADMIN_EMAIL` and let's use `REANA_EMAIL_SENDER` instead?